### DISCUSSION TITLE
[Feature]: Playing two/multiple subtitles at the same time

### DISCUSSION THREAD

**Discussion by wonkyDD** [TheWidlarzGroup/react-native-video#4978] [Ideas]
### What problem are you trying to solve?

There are often situations where I want to view multiple subtitles at once in a mobile environment.

### Proposed solution

Playing two/multiple subtitles at the same time.

Example:

<img width="601" alt="image" src="https://github.com/TheWidlarzGroup/react-native-video/assets/81248624/7012a93c-ed87-46c0-8e18-ebc167f3cd0a">

**Comment by KrzysztofMoch** [comment]
I think is is really cool idea, but does AVPlayer _(iOS)_ and ExoPlayer _(Android)_ support it ?

**Comment by TWG-AI** [comment]
Thanks for the idea — noted for the team to consider.

> I'm an AI assistant, so double-check any specifics I may have mentioned.
